opensrp-anc/src/main/java/org/smartregister/anc/library/activity/ContactJsonFormActivity.java

```diff
@@ -266,7 +266,7 @@ public void proceedToMainContactPage() {
         Contact contact = getContact();
         contact.setJsonForm(ANCFormUtils.addFormDetails(currentJsonState()));
         contact.setContactNumber(contactNo);
-        ANCFormUtils.persistPartial(baseEntityId, getContact());
+        org.smartregister.anc.library.util.ANCFormUtils.persistPartial(baseEntityId, getContact());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'org.smartregister.anc.library.util.ANCFormUtils.persistPartial' due to existing import 'org.smartregister.anc.library.util.ANCFormUtils'](https://app.codacy.com/manual/OpenSRP/opensrp-client-anc/pullRequest?prid=5257516)